site/docs/guides/connect-network.md

```diff
@@ -157,34 +140,17 @@ To allow SSH tunneling to a database instance hosted on Azure, you'll need to cr
 Instructions for Azure Database For PostgreSQL can be found [here](https://docs.microsoft.com/en-us/azure/postgresql/howto-manage-vnet-using-portal);
 note that instructions for other database engines may be different.
 
-4. Find and note the host and port for your capture or materialization endpoint.
-  :::tip
-  For database instances hosted in Azure, you can find the host as Server Name, and the port under Connection Strings (`5432` for PostgreSQL and `3306` for MySql).
-  :::
-
-5. Choose an open port on your localhost from which you'll connect to the SSH server.
-
 ## Configuration
 
 After you've completed the prerequisites, you should have the following parameters:
 
-* **SSH Endpoint** / `sshEndpoint`: the SSH server's hostname, or public IP address, formatted as `ssh://user@hostname[:port]`
-     :::info Hint
+* **SSH Endpoint** / `sshEndpoint`: the remote SSH server's hostname, or public IP address, formatted as `ssh://user@hostname[:port]`
+
      The [SSH default port is 22](https://www.ssh.com/academy/ssh/port).
      Depending on where your server is hosted, you may not be required to specify a port,
      but we recommend specifying `:22` in all cases to ensure a connection can be made.
-     :::
-* **Private Key** / `privateKey`: the contents of the PEM file
-* **Forward Host** / `forwardHost`: the capture or materialization endpoint's host
-* **Forward Port** / `forwardPort`: the capture or materialization endpoint's port
-* **Local Port** / `localPort`: the port on the localhost used to connect to the SSH server
 
-1. Use these to add SSH tunneling to your capture or materialization definition, either by filling in the corresponding fields
-  in a web app, or by working with the YAML directly. Reference the [Connectors](../../concepts/connectors/#connecting-to-endpoints-on-secure-networks) page for a YAML sample.
-
-2. Proxies like SSH are always run on an open port on your localhost,
-so you'll need to re-configure other fields in your capture or materialization definition.
-
-   1. Set the connector's host property to match `localhost` in the SSH configuration.
+* **Private Key** / `privateKey`: the contents of the PEM file
```

Comment:
I know there's a relationship between SSH private keys and PEM files, but if we're being pedantic I don't think all private key files are necessarily PEM files nor that all PEM files are necessarily valid SSH keyfiles.

Perhaps change this to simply "the contents of the SSH private key file"?